plugin/webpack.js

```diff
@@ -6,6 +6,10 @@ var tern = require("../lib/tern");
 require("./commonjs");
 require("./es_modules");
 
+Array.isArray = Array.isArray || function(v) {
```

Comment:
Please don't patch global objects from the library. We can use a local `isArray` variable for this.